src/inspector/inspectorhandler.cpp

```diff
@@ -70,304 +69,269 @@ bool s_pickedItemsSet = false;
+          }
+
+          QPointF pointF = item->mapToScene(QPoint(0, 0));
+          QPoint point = pointF.toPoint();
+          point.rx() += item->width() / 2;
+          point.ry() += item->height() / 2;
+          QTest::mouseClick(item->window(), Qt::LeftButton, Qt::NoModifier,
+                            point);
+          return obj;
+        }},
+    InspectorHandler::InspectorCommand{
+        "click_notification", "Click on a notification", 0,
+        [](InspectorHandler*, const QList<QByteArray>&) {
+          NotificationHandler::instance()->messageClickHandle();
+          return QJsonObject();
+        }},
+
+    InspectorHandler::InspectorCommand{
         "stealurls",
         "Do not open the URLs in browser and expose them via inspector", 0,
         [](InspectorHandler*, const QList<QByteArray>&) {
           s_stealUrls = true;
           return QJsonObject();
         }},
-    InspectorCommand{"mockFreeTrial",
-                     "Force the UI to show 7 day trial on 1 year plan", 0,
-                     [](InspectorHandler*, const QList<QByteArray>&) {
-                       s_mockFreeTrial = true;
-                       return QJsonObject();
-                     }},
-
-    InspectorCommand{"lasturl", "Retrieve the last opened URL", 0,
-                     [](InspectorHandler*, const QList<QByteArray>&) {
-                       QJsonObject obj;
-                       obj["value"] = UrlOpener::instance()->lastUrl();
-                       return obj;
-                     }},
-
-    InspectorCommand{"set_glean_source_tags",
-                     "Set Glean Source Tags (supply a comma seperated list)", 1,
-                     [](InspectorHandler*, const QList<QByteArray>& arguments) {
-                       QStringList tags = QString(arguments[1]).split(',');
-                       emit MozillaVPN::instance()->setGleanSourceTags(tags);
-                       return QJsonObject();
-                     }},
-
-    InspectorCommand{"force_update_check", "Force a version update check", 1,
-                     [](InspectorHandler*, const QList<QByteArray>& arguments) {
-                       s_updateVersion = arguments[1];
-                       MozillaVPN::instance()->releaseMonitor()->runSoon();
-                       return QJsonObject();
-                     }},
-    InspectorCommand{"force_captive_portal_check",
-                     "Force a captive portal check", 0,
-                     [](InspectorHandler*, const QList<QByteArray>&) {
-                       MozillaVPN::instance()
-                           ->captivePortalDetection()
-                           ->detectCaptivePortal();
-                       return QJsonObject();
-                     }},
-
-    InspectorCommand{
-        "force_captive_portal_detection", "Simulate a captive portal detection",
-        0,
+    InspectorHandler::InspectorCommand{
+        "mockFreeTrial", "Force the UI to show 7 day trial on 1 year plan", 0,
```

Comment:
I know this is really just whitespace changes, but this is also the only inspector command that uses camelCase. It's stands out.